sdk/communication/AzureCommunicationChat/Source/Signaling/Events/TrouterEvent.swift

```diff
@@ -47,7 +50,20 @@ public enum TrouterEvent {
     /// - Parameters:
     ///   - chatEventId: The ChatEventId.
     ///   - request: The TrouterRequest that contains the event data.
-    init(chatEventId: ChatEventId, from request: TrouterRequest) throws {
+    init(chatEventId: ChatEventId, from trouterRequest: TrouterRequest? = nil) throws {
```

Comment:
I'd recommend not making `TrouterRequest?` have a `nil` default. You should be force to pass something here, and if it happens to be nil, so be it.